src/blocks/FadeTransition.tsx

```diff
@@ -0,0 +1,56 @@
+import useConnections from './hooks/connections'
+import useProps from './hooks/props'
+
+import { ArgTuple, ComposableProps } from './types'
+
+export const options = {
+  actions: {},
+  hooks: {
+    show: {
+      args: [['args', 'any'] as ArgTuple],
+      returns: 'void',
+    },
+    hide: {
+      args: [['args', 'any'] as ArgTuple],
+      returns: 'void',
+    },
+  },
+  props: {
+    time: 'string',
```

Comment:
Not sure if this needs to be a number or string for a CSS var. I think a number, but I could be wrong.

Reply:
I left it as `string` I didn't work to concatenate it in the css